target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -39,6 +39,10 @@ case "$FIRMWARE" in
 		caldata_extract "0:ART" 0x1000 0x2f20
 		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_ascii 0:APPSBLENV ethaddr) +1)
 		;;
+	askey,rt4230w)
+		caldata_extract "ART" 0x1000 0x3f1f
+		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary ART 0x6) +1)
```

Comment:
No address found in ART 0x1002?

Reply:
The MAC addresses match up with stock fw. Confirmed with ifconfig and WiFi Analyzer on android.